config/web.js

```diff
@@ -111,7 +112,7 @@ const config = {
 
         // Configure how cookies behave in browsers.
         cookieOptions: {
-          password: process.env.COOKIE_PASSWORD,
+          password: process.env.COOKIE_PASSWORD || 'testtesttesttesttesttesttesttest',
```

Comment:
Can you confirm the default is expected here and not a leftover?

Reply:
I did do that on purpose. But now that you ask, I suppose in this particular case we'd prefer to default to something random, juuuust in case it makes it to production. During CI runs, the value doesn't matter because cookie doesn't have to last longer than a unit test (curiously, exporting the `COOKIE_PASSWORD` in the Action didn't seem to work if you look at a couple of the [failed runs](https://github.com/UN-OCHA/hid-api/actions/runs/5080550178/jobs/9127663778)).